### DISCUSSION TITLE
random issues with pull_request / build (ubuntu2004, clang-10) only

### DISCUSSION THREAD

**Discussion by xtcyclist** [vesoft-inc/nebula#4429] [General]
it seems like this 'pull_request / build (ubuntu2004, clang-10) (pull_request)' pipeline often fails with random errors repeatedly, when all the other tests are fine. 

failed tests run ok in dev machines offline, with clang-10.

examples include invalid space vid len, tag not exist, host not enough, ...

is there anything special with this pipeline?

**Comment by HarrisChu** [comment]
we build clang-10 with `DENABLE_ASAN=on`

and sometimes the schema doesn't keep consistent.

**Reply by xtcyclist** [reply]
yes, I suspect there is something wrong with the schema. 

in test case like:
create tag TAG_A ...
wait 6 seconds
stmt_1
stmt_2
stmt_3
...

if for some performance reasons, it takes longer than 6 seconds for TAG_A to be created, the following statements stmt_1 may suffer from a 'tag not exist' error.

my guess only, not sure.

**Reply by xtcyclist** [reply]
6 seconds is already way longer than 2x the configured heartbeat cycle (i.e., 1s). seem long enough.
